public/app/plugins/datasource/alertmanager/ConfigEditor.tsx

```diff
@@ -51,6 +52,7 @@ export const ConfigEditor: React.FC<Props> = ({ options, onOptionsChange }) => {
         dataSourceConfig={options}
         showAccessOptions={true}
         onChange={onOptionsChange}
+        sigV4AuthToggleEnabled={config.sigV4AuthEnabled}
```

Comment:
Do you need:
```jsx
renderSigV4Editor={<SIGV4ConnectionConfig {...props}></SIGV4ConnectionConfig>}
```
the same way that it's used in the prometheus and elasticsearch datasource plugins?